### ISSUE TITLE
Closed captions are not available for the video content.: A11y_Aspire website_Community Videos_Captions (Pre-recorded)

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/aspire.dev#237]
### GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev1; #A11yWCAG; #DesktopWeb; #ChromiumEdge; #Win11; #Developer Community Partners; #SH_AspireWebsite_Web_Jan26; #WCAG1.2.2; #Captions (Pre-recorded);

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."**

### Environment Details:
Application Name: Aspire website
URL: https://aspire.dev/docs/
Chromium Edge: Version 145.0.3775.0 (Official build) dev (64-bit)
Edition: Windows 11 Enterprise Insider Preview
Version: 24H2
OS build: 26540.1001

### Repro steps:

1. Hit the URL: https://aspire.dev/community/videos/.
2. TAB to video present under “Vimeo" and hit ENTER key.
3. Observe the issue that Closed captions are not available for the video content.

### Actual Result:
No Closed Captions (CC) option is available in the video player. Audio information is delivered only through sound.

### Expected Result:
Closed captions should be provided for all prerecorded video content. Captions should be synchronized with the audio and easy to discover through the video controls.

### User Impact:
Deaf and hard‑of‑hearing users cannot access audio information in the video. Users who depend on captions to understand speech and sound cues are blocked from consuming the content.

### Attachment:

https://github.com/user-attachments/assets/335b9970-846f-477b-be09-0dd3209543f9
